Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -97,38 +120,57 @@ There are no known security impacts of the features in this specification.
-
-* Why not some combination of all the above?
-    * [Pro] Allows for the greatest flexibility with multiple paths to achieve the same goal, so authors can choose the approach that best fits their use case or preference.
-    * [Con] More complex backend implementation and less obvious frontend implementation due to the combination available to specify per-document or per-element `handwriting` state.
-
-* Why not extend a different attribute or property?
-    * [HTML] `inputmode`
-        * Related to but distinct from &lt;input&gt; `type`. Is only concerned with virtual keyboard inputs.
-    * [CSS] `pointer-events`:
-        * Is concerned with whether an element or visual components of an element can be the target of a pointer event, not what kinds of pointer devices can be used.
+### Why not a JavaScript API?
+(e.g., `e.enableHandwriting()` or `e.setHandwritingState(...)`, `e.getHandwritingState()` and `e.getComputedHandwritingValue()`)
+
+ * [Pro] Granular control over which elements should allow handwriting input.
+ * [Con] Introduces more complexity and is not as simple as an HTML attribute or CSS property.
+ * [Con] A CSS property can be exposed to JavaScript as a IDL attribute which may be more ergonomic.
+ * [Con] If users, touch  and pen, or organizations disable JavaScript for their browsers there would need to be another mechanism to disable handwriting input.
+
+### Why not some combination of all the above?
+* [Pro] Allows for the greatest flexibility with multiple paths to achieve the same goal, so authors can choose the approach that best fits their use case or preference.
+* [Con] More complex backend implementation and less obvious frontend implementation due to the combination available to specify per-document or per-element `handwriting` state.
+
+### Why not extend a different attribute or property?
+[HTML]  [inputmode](https://www.w3.org/TR/2002/WD-xforms-20020821/sliceE.html):
+* Related to but distinct from `<input>`  `type`. Is only concerned with virtual keyboard inputs.
+[CSS]  [pointer-events](https://w3c.github.io/pointerevents/#pointerevent-interface):
+* Is concerned with whether an element or visual components of an element can be the target of a pointer event, not what kinds of pointer devices can be used.
+
+## References and acknowledgements
+
+* @**[flackr](https://github.com/flackr)**, @**[mustaqahmed](https://github.com/mustaqahmed)**, @**[adettenb](https://github.com/adettenb)**, @**[patrickhlauke](https://github.com/patrickhlauke)**, @**[ogerchikov](https://github.com/ogerchikov)** for carrying forward the discussion.
+* Previous explainer: [Handwriting Explainer](Handwriting/explainer.md)
+* Existing discussion: https://github.com/w3c/pointerevents/issues/516, https://www.w3.org/2024/11/06-pointerevents-minutes.html
+
+## Stakeholder Feedback / Opposition
+
+None other than the discussion linked in the **References and Acknowledgements** section
```

Comment:
I think it's worth enumerating a few of these as well as referencing the conversations:

* `touch-action` in its current state may not be flexible enough for developers needs.
    * The property name, while clearly communicated in the Spec, isn't specific to touch behaviors as it includes stylus/pen actions as well.
    * Developers may want granular control over input type in addition to "actions".
    * Developers may want granular control to specify "action" precedence (handwriting then scrolling, or vice versa).
* Concerns with how handwriting and panning can intuitively co-exist, since it's possible a scrollable page with `touch-action: handwriting pan-y` may be unable to be panned. e.g., when the entire document is editable.